cinema/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+from .models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=100)
+    description = serializers.CharField(max_length=255)
```

Comment:
The `description` field is defined with a maximum length of 255 in the serializer. Ensure that this matches the model's field definition. If you change the model's `description` field to `TextField` as suggested earlier, you should also update this serializer field accordingly.